src/Components/Feelings/feelingsData.js

```diff
@@ -0,0 +1,59 @@
+const feelings = [{
+    'feeling': 'Happy',
+    'associatedFeelings': [
+      'Happy',
+      'Content',
+      'Optimistic'
+    ]
+  },
+  {
+    'feeling': 'Sad',
+    'associatedFeelings': [
+      'Sad',
+      'Lonely',
+      'Depressed'
+    ]
+  },
+  {
+    'feeling': 'Surprised',
+    'associatedFeelings': [
+      'Surprised',
+      'Confused',
+      'Excited'
+    ]
+  },
+  {
+    'feeling': 'Bad',
+    'associatedFeelings': [
+      'Bad',
+      'Stressed',
+      'Tired'
+    ]
+  },
+  {
+    'feeling': 'Fearful',
+    'associatedFeelings': [
+      'Fearful',
+      'Anxious',
+      'Insecure'
+    ]
+  },
+  {
+    'feeling': 'Angry',
+    'associatedFeelings': [
+      'Angry',
+      'Distant',
+      'Frustrated'
+    ]
+  },
+  {
+    'feeling': 'Disgusted',
+    'associatedFeelings': [
+      'Disgusted',
+      'Disappointed',
+      'Awful'
+    ]
+  }
+];
+
+export default feelings;
```

Comment:
Love this organization!

Reply:
Plus one, so easy to read and understand our schema.

Reply:
+++

Thank you so much for cleaning this up! Super readable and straightforward!